drapps/create.py

```diff
@@ -358,6 +380,19 @@ def parse_env_vars(ctx, param, value):
     type=click.Path(exists=True, dir_okay=False, resolve_path=True, path_type=Path),
     help='Path to tar archive with custom application docker images.',
 )
+@click.option(
+    '--replicas',
+    required=False,
+    type=click.INT,
+    default=1,
+    help='Number of replicas to be created.',
+)
+@click.option(
+    '--cpu-size',
+    required=False,
+    type=click.Choice(['nano', 'micro', 'small', 'medium', 'large', 'xlarge', '2xlarge']),
+    default='small',
```

Comment:
```suggestion
    default='small',
    help=(
        "CPU size table:\n"
        "- nano: 1 CPU, 128MB\n"
        "- micro: 1 CPU, 256MB\n"
        "- small: 1 CPU, 512MB\n"
        "- medium: 1 CPU, 1GB\n"
        "- large: 2 CPU, 1.5GB\n"
        "- xlarge: 2 CPU, 2GB\n"
        "- 2xlarge: 2 CPU, 3GB"
    )
```
Not sure if it looks right in the help text like this, but would be good to include it somewhere. If not here then maybe a small addition in the README :thinking:

Reply:
nit: in the UI we call nano and micro xxs and xs, let's try to stay consistent

![image](https://github.com/user-attachments/assets/6abb19b0-dcc3-4b4c-9f2d-6dd06966fe7e)